cibyl/outputs/cli/ci/system/impls/jobs/colored.py

```diff
@@ -35,23 +35,20 @@ class ColoredJobsSystemPrinter(ColoredPrinter, CISystemPrinter):
     def print_system(self, system, indent=0):
         printer = IndentedTextBuilder()
 
-        printer.add(self.palette.blue('System: '), indent)
-        printer[-1].append(system.name.value)
-
-        if self.verbosity > 0:
-            printer[-1].append(f' (type: {system.system_type.value})')
+        # Begin with the text common to all systems
+        printer.add(super().print_system(system, indent=indent), indent)
 
         if self.query != QueryType.NONE:
             for job in system.jobs.values():
                 printer.add(self.print_job(job), indent+1)
 
-            if system.is_queried():
+            if not system.is_queried():
+                printer.add(self.palette.blue('No query performed'), indent+1)
+            elif system.is_queried() and self.query != QueryType.FEATURES:
```

Comment:
'system.is_queried()' is going to be true if it gets here.

Reply:
Good catch, thanks!